Map/Test/MapTest.cpp

```diff
@@ -155,8 +142,52 @@ TEST_F(SquareMapTest, treasureFieldInvisiblePrint)
   ASSERT_EQ(' ',map.printField(field));
 }
 
-TEST_F(SquareMapTest, wallFieldInvisiblePrint)
+TEST_F(SquareMapTest, wallFieldVisiblePrint)
 {
-  WallField field;
-  ASSERT_EQ(' ',map.printField(field));
+    WallField field;
+    ASSERT_EQ('#', map.printField(field));
+}
+
+TEST_F(SquareMapTest, doorFieldVisiblePrint)
+{
+    DoorField field;
+    ASSERT_EQ('D', map.printField(field));
+}
+
+// Visilibity Updater
+
+TEST_F(SquareMapTest, roomShouldBeInvisibleByDefault)
+{
+    SquareMap map(mapSize);
+    constexpr int expectedNumberOfVisibleFields = 0;
+    auto &fieldList = map.getFields();
+
+    int numberOfVisibleFields = 0;
+
+    for(const auto& column : fieldList)
+    {
+        numberOfVisibleFields += std::count_if(column.begin(), column.end(),
+            [&](const auto &field){ return field->isVisible(); });
+    }
+
+    ASSERT_EQ(expectedNumberOfVisibleFields, numberOfVisibleFields);
+}
+
+TEST_F(SquareMapTest, roomShouldBeVisibleAfterUpdate)
```

Comment:
Fajnie by bylo jakbyś stworzył jakas mape (recznie wypelnic ja odpowiednimi polami), np 3x3 ze scianami  i drzwiami, zeby bylo widac ze jest widoczna tylko czesc mapy, a jak wejdziesz na drzwi to sie reszta pokazuje